.storybook/reactIntl.ts

```diff
@@ -1,21 +1,19 @@
-import enUS from '../i18n/en-US';
-import deDE from '../i18n/de-DE';
-import frFR from '../i18n/fr-FR';
-import jaJP from '../i18n/ja-JP';
-import zhCN from '../i18n/zh-CN';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const languages = require('@box/languages');
```

Comment:
tiny suggestion: to make it more consistent as how we declare this variable in story/preview.tsx, can we update to `const boxLanguages = require('@box/languages');`